pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -158,12 +158,15 @@ def cmd(self):
 
         if self.disable_capture:
             cmd.append("-s")
-
         if self.xdist_ip_addresses:
             cmd.append('--dist=loadscope')
-            for ip in self.xdist_ip_addresses.split(' '):
-                xdist_string = '--tx ssh=ubuntu@{}//python="source /edx/app/edxapp/edxapp_env; ' \
-                               'python"//chdir="/edx/app/edxapp/edx-platform"'.format(ip)
+            for ip in self.xdist_ip_addresses.split(','):
+                # The django settings runtime command does not propagate to xdist remote workers
+                django_env_var_cmd = 'export DJANGO_SETTINGS_MODULE={}' \
+                                     .format('{}.envs.{}'.format(self.root, self.settings))
+                xdist_string = '--tx {}*ssh="ubuntu@{} -o StrictHostKeyChecking=no"' \
+                               '//python="source /edx/app/edxapp/edxapp_env; {}; python"' \
+                               '//chdir="/edx/app/edxapp/edx-platform"'.format(self.processes, ip, django_env_var_cmd)
```

Comment:
It looks like this command will break with the default `self.processes` value of `0`.